extended/src/main/java/io/kubernetes/client/extended/kubectl/util/selector/SelectorHelper.java

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package io.kubernetes.client.extended.kubectl.util.selector;
+
+import io.kubernetes.client.openapi.models.V1LabelSelector;
+import io.kubernetes.client.openapi.models.V1LabelSelectorRequirement;
+import io.kubernetes.client.util.labels.EqualityMatcher;
+import io.kubernetes.client.util.labels.LabelMatcher;
+import io.kubernetes.client.util.labels.LabelSelector;
+import io.kubernetes.client.util.labels.SetMatcher;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class SelectorHelper {
+  protected static final String LabelSelectorOpIn = "In";
+  protected static final String LabelSelectorOpNotIn = "NotIn";
+  protected static final String LabelSelectorOpExists = "Exists";
+  protected static final String LabelSelectorOpDoesNotExist = "DoesNotExist";
+
+  public static LabelSelector LabelSelectorAsSelector(V1LabelSelector ps) throws Exception {
+    int matcherNum = 0;
+    if (ps.getMatchLabels() != null) matcherNum = ps.getMatchLabels().size();
+    if (ps.getMatchExpressions() != null) matcherNum = ps.getMatchExpressions().size();
+
+    if (matcherNum == 0) {
+      return LabelSelector.empty();
+    }
```

Comment:
if i'm reading correctly, it doesnt make a difference if L32-38 are removed, is it?